sonaranalyzer-dotnet/its/expected/Ember-MM/Ember.Plugins-{9496C697-5AFD-4813-AEDC-AF33FACEADF0}-S4058.json

```diff
@@ -0,0 +1,69 @@
+{
+"issues":  [
+{
+"id":  "S4058",
+"message":  "Change this call to 'lTitle.EndsWith' to an overload that accepts a 'StringComparison' as a parameter.",
```

Comment:
I like it, as it gives more context. But it's not what we normally do so it might be confusing in some cases and won't be consistent. In this very case, I'm fine with either.

Perhaps we should start adding full expression in all messages? @Evangelink  @valhristov @jccollet WDYT?